tempus_fugit_minecraft/player.py

```diff
@@ -275,9 +276,9 @@ def update(self, dt: float, collision_checker: Callable[[tuple, int], tuple]) ->
             # Update your vertical speed: if you are falling, speed up
             # until you hit terminal velocity; if you are jumping, slow
             # down until you start falling.
-            self.dy -= dt * self.GRAVITY
-            self.dy = max(self.dy, -self.MAX_FALL_SPEED)
-            dy += self.dy * dt
+            self.vertical_velocity_in_blocks_per_second -= dt * self.GRAVITY_IN_BLOCKS_PER_SECOND_SQUARED
+            self.vertical_velocity_in_blocks_per_second = max(self.vertical_velocity_in_blocks_per_second, -self.MAX_FALL_SPEED_IN_BLOCKS_PER_SECOND)
+            dy += self.vertical_velocity_in_blocks_per_second * dt
```

Comment:
In regards to dt, shouldn't this be changed to seconds? Especially since we are essentially using a velocity formula.

Reply:
I named it `delta_time_in_seconds`.